### PR TITLE
Adding new voices naturalness and watermark

### PR DESCRIPTION
**Pull request by springcoil**
@GetOn4 Let me know when you get the voices deployed then we can merge this. Has watermark as well.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,12 +8,16 @@ v0.6.0
...
  ```
+In the future we will add other endFormats - please tell us which ones you'd like. 
 
 * As a developer user I want to share my audio with my team/bosses/business person so they can try and test the wonders of api.audio. We call this *virality link* and it's available in the console now. 
 
+* To continue to be able to offer free content to those of you still getting to know API.audio, we have added a watermark to our files.
 
-### Enterprise features
-* For our enterprise users we've enabled `sandboxing` this allows you to test safely API requests without using up credits. Please contact your account manager for further details. 
+* Enhancements of voices. We've rolled out to some users a more natural pauses to our custom voices. We contacted customers affected and some requested to not have this feature enabled please contact us if you want to use the old voices. We do feel that these voices are more natural and our beta testing was positive. These are all available under `msnr` voices in our API. Let us know what you think.
```
Comment (GetOn4) [inline CHANGELOG.md:17]: Waiting for ms-spech to be in production with the change I need.

**Review by GetOn4 — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (springcoil) [reply CHANGELOG.md:17]: Ok waiting. I'll not merge yet.

**Review by guillem-211 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (springcoil) [reply CHANGELOG.md:17]: @GetOn4 Give me a headsup when speech is deployed

**Pull request merged**